.github/workflows/integration_tests.yml

```diff
@@ -0,0 +1,28 @@
+---
+name: integration_tests
+on:
+  push:
+    tags:
+      - v*
+    branches:
+      - master
+      - main
+  pull_request:
+permissions:
+  contents: read
+  # Optional: allow read access to pull request. Use with `only-new-issues` option.
+  # pull-requests: read
+jobs:
+  test:
+    name: lint
```

Comment:
`lint` does not define which integration test it is running, maybe we can rename this to something more specific?

Reply:
It's a miss from my side. Fixed! THansk for flagging